loops-arrays-objects.js

```diff
@@ -131,5 +163,57 @@ const students = [
 // ]
 // ==========================================
 
```

Comment:
Of met Switch. Jouw uitwerking is ook prima.
https://www.w3schools.com/js/js_switch.asp